app/components/access_panels/other_listings_component.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+module AccessPanels
+  class OtherListingsComponent < AccessPanels::Base
+    def render?
+      earthworks_url.present? || purls.present?
+    end
+
+    def earthworks_url
+      @earthworks_url ||= construct_earthworks_url
+    end
+
+    # @return [Array<String>] a tags representing PURL links
+    def purl_links
+      @purl_links ||=
+        if purls.size > 1 # text of the link is the actual URL if there is more than one unique URL, so there aren't multiple links with the same title
+          purls.map { |purl| link_to(purl, purl) }
+        elsif purls.size == 1
+          Array(link_to('Stanford Digital Repository', purls.first))
+        else
+          []
+        end
+    end
+
+    private
+
+    # @return [Array<String>] URLs representing links to PURL, whether specified in MARC or MODS, and de-duped, e.g. ["https://purl.stanford.edu/hj948rn6493"]
+    def purls
+      @purls ||= (mods_purls + marc_managed_purls).uniq
+    end
+
+    # @return [Array<String>] URLs representing links to other locations, extracted from MODS metadata, e.g. ["https://purl.stanford.edu/hj948rn6493"]
+    def mods_purls
+      Array(document['url_fulltext']).find_all { |str| str.start_with?(Settings.PURL_EMBED_RESOURCE) }
```

Comment:
i wasn't sure if `document['url_fulltext']` was the right field to use.  other possible options i saw were:
* `document.as_schema_dot_org['identifier']` (which seems to have an array of PURLs, but i ended up moving away form using `as_schema_dot_org` at all, so guessing we don't want to reintroduce its usage here)
* `document.mods&.location&.map { |location| location.values }&.flatten` (in testing, this got me an array of `a` tags for the PURLs, but the text of the tag was the URL, which didn't match the mockup's language, and parsing the PURL out of the tag seemed unnecessarily indirect)